lib/src/rules/avoid_print.dart

```diff
@@ -14,13 +14,45 @@ const _desc = r'Avoid `print` calls in production code.';
 const _details = r'''
 **DO** avoid `print` calls in production code.
 
+For production code use `debugPrint`, a logging framework, or surround `print`
+calls with a check for `kDebugMode`.
```

Comment:
I like the idea of mentioning these alternatives, but these only exist for Flutter users and a Dart developer may be confused when they cannot find these.

Maybe we can make that explicit somehow?

```suggestion
For production code, consider using a logging framework.
If you are using Flutter, you can also use use `debugPrint`
or surround `print` calls with a check for `kDebugMode`
```

Reply:
Very good point, looks better too! :+1: